echopype/clean/transient_noise/transient_matecho.py

```diff
@@ -123,13 +123,29 @@ def transient_noise_matecho(
     min_window: float = 20,
 ) -> xr.DataArray:
     """
-    Build a Matecho-style (adapted from Matecho) transient-noise mask from an xarray Dataset.
+    Matecho-style transient-noise mask (column-wise).
 
-    This wrapper prepares the vertical coordinate and optional bottom, then calls a
-    NumPy core via `apply_ufunc`, vectorized across leading dims (e.g., channel).
-    The method flags entire ping columns as transient when the ping’s mean Sv
-    (computed in linear units) exceeds a local percentile + delta_db within a
-    deep window.
+    Overview
+    --------
+    Flags entire pings as transient when, within a deep window, the ping’s
+    mean Sv (computed in linear units, then converted back to dB) exceeds a
+    local reference percentile by `delta_db`.
+
+    1) Deep window: Use a vertical slice from `start_depth` to
+       `start_depth + window_meter`, limited by a local bottom (if provided;
```

Comment:
with the `start_depth` to `start_depth + window_meter` specification, this does not have to be "deep" per se, right? How about changing it to just "Depth window"?